src/librustc_mir/interpret/snapshot.rs

```diff
@@ -1,3 +1,17 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
copyright comments are not necessary anymore

Reply:
Oh ok, I thought I had missed that the first time, I'll amend this evening.